### PR TITLE
Feature/#276 proxy subdomain

### PR DESCRIPTION
**Pull request by AlviseSembenico**
Here we go, I added the feature to strip the url in order to fetch some resource correctly but I am not sure if it is how it is meant to be, we can discuss it on Monday.
Thanks

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -5,6 +5,7 @@
 #    pip-compile --output-file requirements/base.txt requirements/base.in
 #
 -e git+https://github.com/maykinmedia/gemma-zds-client-3.4.git@e826a2fa111100d33d7c164a7bdf95d07c3b916c#egg=gemma-zds-client
+-e git+https://github.com/maykinmedia/django-subdomains.git#egg=django-subdomains
```
Comment (alextreme) [inline requirements/base.txt]: @AlviseSembenico ensure you pin this against a commit, or make the use of master explicit if you're brave

**Comment by alextreme** [comment]
@AlviseSembenico could you get this working with subdomains of the vorm:

proxy1234-vng-staging.maykin.nl ?

*.maykin.nl is already pointing to rachel, and if you see in deployments/nginx-*.conf we have a SSL wildcard certificate installed. So we'd only have to handle all servers with a specific postfix (if Nginx allows this, I'm not sure)

We can also get a certificate for proxy1234.vng-staging.maykin.nl if necessary but from experience DNS wildcards at multiple levels will get tricky.

**Comment by AlviseSembenico** [comment]
I tryied to deploy it in the test but clearly we have to unlock it othewise the authentication is asked by nginx

**Comment by AlviseSembenico** [comment]
@alextreme Everything operative

**Comment by alextreme** [comment]
@AlviseSembenico great! Feel free to roll this out on vng-test

**Pull request merged**